### ISSUE TITLE
[Limbs System] Hands

### ISSUE THREAD

**Issue by Swagile** [HippieStationCode/HippieStation13#3024]
Problem Description: What is the problem?

What did you expect to happen: Why do you think this is a bug?

What happened instead: How is what happened different from what you expected?

Why is this bad/What are the consequences: Why do you think this is an important issue?

Steps to reproduce the problem: The most important section. Review everything you did leading up to causing the issue.

Server: Hippie

Revision: Latest Hippie codebase.

Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?): Double Agents. I was running around waiting for a admin to respond on my TC trade request when I got close to a person who got PDA bombed. I lost one leg and both my arms which ended in a good medic healing me up. While I was waiting for my TC trade to fully go through in a secluded area, I started trying to play around with my items. Thats when I figured out that even with BOTH my hands gone, I could still grab my PDA out of the PDA slot and keep it in hand. I could also do this with a backpack if I click held and dropped onto my hand slot. I can also check my backpack / box by click holding onto my sprite which completely bypasses the usual checks made on backpack and box usage aka clicking on them manually.

TL;DR certain items can still be held / interacted with even without both hands.

**Comment by JohnGinnane** [comment]
Can you recall if when your arms were blown off the hand slots were cross out? Could you pick up items that were not on your body?

**Comment by ArcLumin** [comment]
I'm assuming anything that you can drag equip to do this. I'll need to test it.

**Comment by Swagile** [comment]
Both hands were crossed out with a big fat O with a thingy over it, so yes, my hands were completely fucked.

My sprite also showed both hands missing as nothing was around the hand area; it was just empty.

I did not check whether I could pick up items off the floor but I am sure if I drag and dropped a backpack onto my own backpack slot id be able to equip it. Not sure how to test this though.

**Comment by JohnGinnane** [comment]
Might have to blown our arms off and then try drag items onto our sprite to see what happens

**Comment by Swagile** [comment]
I do not know how to force compile past mundane errors like the fact that our FPS proc overrides a game proc and because of this my compile stops so until I fix that I am unable to test this more than what I did ingame yesterday.

**Comment by JohnGinnane** [comment]
Have you updated your master?

**Comment by McBawbaggings** [comment]
Change from BETA to STABLE

**Comment by Swagile** [comment]
Yeah I am on the BETA because my BYOND fucks up in the Escape Shuttle unless I am in BETA.

Ill see if stable fixes it.

**Comment by ArcLumin** [comment]
Okay, after testing, I can confirm some things;

1. Having no arms means that you can, for a single tick, put something in your hands. However, you immediately drop it once you do so. 
2. The PDA is still perfectly usable without arms. Due to it working without needing to be in your hands, having no hands means nothing to it. Simply drag it to your character and you can use it perfectly fine.

**Comment by ArcLumin** [comment]
You can also right click to toggle your jetpack on or off with no hands. Gimpy flight!

**Comment by JohnGinnane** [comment]
Thanks for the testing. I wrote the code to handle when you had something in your hands but didn't actually have hands: https://github.com/HippieStationCode/HippieStation13/blob/master/code/modules/mob/living/carbon/human/inventory.dm#L338

This means yeah, for one tick you could hold something. Perhaps we could make it so you can't even pick stuff up with no hands just to be sure.

For PDA we should definitely check for no hands on interact

**Comment by McBawbaggings** [comment]
You've always been able to do some actions with both of your hands full, this is an extension of that.
